feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -76,7 +76,8 @@
               cxModal="dismiss"
               cxModalReason="View Cart click"
               class="btn btn-primary"
-              autofocus
+              (esc)="dismissModal('ESC pressed')"
```

Comment:
> Opened modals should allow to close them on ESC key press.

1 I can not close import dialog by escape after upload (when seeing summary) - need to click the mouse on modal first or press tab first (but it will work only if `show more` button exists)

I think trigger focus to the element is necessary when finished.

2 I can still use escape while uploading when click on modal first (or press tab if `show more` button exists)

something like this could help for point 2:
![image](https://user-images.githubusercontent.com/8040837/159805966-9194a80d-9398-48f6-8e88-4595be1437bb.png)